tests/integration/source_code/test_jobs.py

```diff
@@ -112,28 +93,24 @@ def test_job_task_linter_library_installed_cluster(
     make_random,
     make_cluster,
     make_notebook,
-    make_directory,
 ) -> None:
     created_cluster = make_cluster(single_node=True)
-    libraries_api = ws.libraries
-    libraries_api.install(created_cluster.cluster_id, [Library(pypi=PythonPyPiLibrary("greenlet"))])
-    entrypoint = make_directory()
+    ws.libraries.install(created_cluster.cluster_id, [Library(pypi=PythonPyPiLibrary("dbt-core==1.8.7"))])
 
-    notebook = make_notebook(path=f"{entrypoint}/notebook.ipynb", content=b"import doesnotexist;import greenlet")
+    notebook = make_notebook(content=b"import doesnotexist;import dbt")
```

Comment:
what happens if `dbt` gets to the known list? would we have to update all the tests again? i wonder if we can add an irrelevant library, like https://pypi.org/project/Adyen/ (i don't think that anyone will ever call payment APIs from notebooks) or https://pypi.org/project/emoji

Reply:
I like the emoji project!

I added dbt because that is (usually) not installed on a runtime, it is ran from a laptop sending queries to databricks over an endpoint

Reply:
we have a dbt job task type, so i wonder. let's keep dbt for now then.